pmd-java/src/main/java/net/sourceforge/pmd/lang/java/typeresolution/MethodType.java

```diff
@@ -29,11 +29,15 @@ public JavaTypeDefinition getReturnType() {
         return returnType;
     }
 
-    public List<JavaTypeDefinition> getArgTypes() {
+    public List<JavaTypeDefinition> getParameterTypes() {
         return argTypes;
     }
 
     public Method getMethod() {
         return method;
     }
+
+    public JavaTypeDefinition getVarargComponentType() {
```

Comment:
seems that this really needs a companion method `isVararg()` and maybe even throw if not vararg